src/chrome/chromeDebugAdapter.ts

```diff
@@ -646,75 +655,88 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
         this._lineColTransformer.columnBreakpointsEnabled = this._columnBreakpointsEnabled;
     }
 
-    protected async onScriptParsed(script: Crdp.Debugger.ScriptParsedEvent): Promise<void> {
-        this.doAfterProcessingSourceEvents(async () => { // This will block future 'removed' source events, until this processing has been completed
-            if (typeof this._columnBreakpointsEnabled === 'undefined') {
-                await this.detectColumnBreakpointSupport(script.scriptId).then(async () => {
-                    await this.sendInitializedEvent();
-                });
-            }
-
-            if (this._earlyScripts) {
-                this._earlyScripts.push(script);
-            } else {
-                await this.sendLoadedSourceEvent(script);
-            }
-        });
-
-        if (script.url) {
-            script.url = utils.fixDriveLetter(script.url);
+    public breakpointsResolvedDefer(scriptId: string): PromiseDefer<void> {
```

Comment:
What's the verb in this method name, should it be a "get"?

Reply:
getBreakpointsResolvedDefer